Added Components

Pradhumn07 opened the pull request.

Pradhumn07 committed:
Added Components

instagagan commented on src/components/DataList.tsx:
```diff
@@ -0,0 +1,35 @@
+        className={outlined ? OUTLINED_INPUT : CONTAINED_INPUT}
+      />
+      <datalist id="datalistOptions">
+        <option value="Mumbai"/>
```
No hardcoded value in components

instagagan reviewed (changes requested):
- Add Comments, more appropriately.
- ClassName argument is missing from all the components.
- Format your code, use visual studio code extension Prettier.
- code is not responsive
- No hardcoded value in components

Pradhumn07 committed:
all changes done accept responsive

I am working to make components responsive but there are some error while doing so it is taking some time

gagandeep commented on package.json:
```diff
@@ -44,13 +45,13 @@
     "@storybook/addons": "^5.3.18",
     "@storybook/preset-create-react-app": "^2.1.1",
-    "@storybook/react": "^5.3.18",
+    "@storybook/react": "^6.1.21",
```
This is the problem with your setup.

Pradhumn07 committed:
Merge branch 'master' of https://github.com/innerkorehq/innerkore-design-system
